opm/simulators/flow/Main.hpp

```diff
@@ -20,7 +20,7 @@
   You should have received a copy of the GNU General Public License
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
-#ifndef OPM_MAIN_HEADER_INCLUDED
+#ifndef OPM_MAIN_HEADER_INCLUDEDp
```

Comment:
This&ndash;the trailing `p` on `_INCLUDED`&ndash;may break something, although I can't think of anywhere we would include `Main.hpp` twice.

Reply:
Sorry, sloppy fingers.